circuits-circom/venmo_send.circom

```diff
@@ -15,11 +15,11 @@ template VenmoSendEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
     assert(n * k > 1024); // constraints for 1024 bit RSA
 
     // Rounded to the nearest multiple of pack_size for extra room in case of change of constants
-    var max_email_amount_len = 7; // Allowing max 4 fig amount + one decimal point + 2 decimal places
+    var max_email_amount_len = 8; // Allowing max 5 fig amount + one decimal point + 2 decimal places. e.g. $2,500.00
```

Comment:
```javascript
var max_email_amount_len = 8; // Allowing max 4 fig amount + one comma + one decimal point + 2 decimal places. Max allowed amount $9,999.00 is above the current Venmo limit of $4,999.00
```


The maximum amount that can be sent via Venmo to another user in a P2P payment is $4,999.99 per transaction, with the same amount as the maximum weekly limit for such payments. [Source](https://wise.com/us/blog/venmo-transfer-limit#:~:text=Please%20remember%20that%20for%20person,Take%20a%20look%20at%20Wise)

Reply:
Fixed https://github.com/zkp2p/zk-p2p/pull/131/files